coduno.go

```diff
@@ -89,7 +157,16 @@ func cors(w http.ResponseWriter, req *http.Request) {
 
 	// only allow CORS on localhost for development
 	if strings.HasPrefix(origin, "http://localhost") {
+		// The cookie related headers are used for the api requests authentication
+		w.Header().Set("Access-Control-Allow-Methods", "OPTIONS,GET,POST,PUT,DELETE")
+		w.Header().Set("Access-Control-Allow-Headers", "cookie,content-type")
+		w.Header().Set("Access-Control-Allow-Credentials", "true")
 		w.Header().Set("Access-Control-Allow-Origin", origin)
+		w.Header().Set("Access-Control-Expose-Headers", "Access-Control-Allow-Origin,set-cookie")
```

Comment:
Looking at [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#Access-Control-Expose-Headers) and [that](https://quickleft.com/blog/cookies-with-my-cors/) I think we can drop that header.